internal/layer2/announcer.go

```diff
@@ -140,33 +155,37 @@ func (a *Announce) updateInterfaces() {
 
 func (a *Announce) spamLoop() {
 	// Map IP to spam stop time.
-	m := map[string]time.Time{}
+	type spamInfo struct {
```

Comment:
I'd call this `timedSpam` or `deferredSpam`.

Reply:
ok